src/gebt_poc/solver.cpp

```diff
@@ -16,22 +16,46 @@ UserDefinedQuadrature::UserDefinedQuadrature(
 }
 
 void Interpolate(
+    Kokkos::View<double*> nodal_values, Kokkos::View<double*> interpolation_function,
+    Kokkos::View<double*> interpolated_values
+) {
+    const auto n_nodes = nodal_values.extent(0) / kNumberOfLieAlgebraComponents;
+    KokkosBlas::fill(interpolated_values, 0.);
```

Comment:
We can use `KokkosBlas::axpby` instead of first filling interpolated values with 0.